app/manifest.json

```diff
@@ -51,5 +55,10 @@
     "fonts/font-awesome/*",
     "dict/*"
   ],
-  "content_security_policy": "script-src 'self' https://www.brcdn.org https://brcdn.org; object-src 'self'"
+  "content_security_policy": "script-src 'self' https://www.brcdn.org https://brcdn.org; object-src 'self'",
+  "__moz__applications": {
+    "gecko": {
+      "id": "textlint@example.com"
```

Comment:
This ID is a dummy.
It should be replaced with more appropriate one when we support Firefox officially and [submit it to AMO](https://developer.mozilla.org/en-US/Add-ons/Distribution).